src/mixins/resource-lookup-mixin.ts

```diff
@@ -27,6 +27,7 @@ export default class ResourceLookupMixin extends Vue {
 
     /**
      * Returns the current entity's full display name.
+     * Used by Correction component.
```

Comment:
We really should name the resource objects in a way that denotes what they're used for. Future.